lib/ansible/modules/cloud/docker/docker_node.py

```diff
@@ -47,9 +51,13 @@
         type: str
     labels_to_remove:
         description:
-            - List of labels that will be removed from the node configuration. If label specified on the list is
-              not assigned to the node then no action will be performed. The remaining labels assigned to the
-              node remains unchanged. The list have to contain only label names, not their values.
+            - List of labels that will be removed from the node configuration. The list has to contain only label
+              names, not their values.
+            - If the label provided on the list is not assigned to the node, the entry is ignored.
+            - If the label is both on the I(labels_to_remove) and I(labels), then value provided in I(labels) remains
+              assigned to the node.
```

Comment:
List of labels may be an output of other task or external script. Changing labels assigned to the node usually happens when you do maintenance or emergency operations and in such cases, it is better if one operation (assigning/modifying label) supersede other (removing labels) - module will fail if there is a problem in reading/writing to swarm manager/node (or typo in option). If for someone label removal is more important than preserving it in such situation then the task can be easily split into two separate tasks - first `labels` then `labels_to_remove`
The module may fail here of course, but that means more difficult error handling in the playbook.

Reply:
Sounds ok. How about still emitting a warning in case a label shows up in both `labels` and `labels_to_remove`? That way users know that something isn't right (and they can investigate), but can also choose to ignore it.